instrumentation-api-incubator/src/main/java/io/opentelemetry/instrumentation/api/incubator/semconv/db/DbClientCommonAttributesGetter.java

```diff
@@ -44,4 +44,20 @@ default String getDbNamespace(REQUEST request) {
   @Deprecated
   @Nullable
   String getConnectionString(REQUEST request);
+
+  @Nullable
+  default String getResponseStatusFromException(Throwable throwable) {
+    return null;
+  }
+
+  @Nullable
+  default String getResponseStatus(RESPONSE response) {
+    return null;
+  }
```

Comment:
`HttpCommonAttributesGetter` has `Integer getHttpResponseStatusCode(REQUEST request, RESPONSE response, @Nullable Throwable error);`

Reply:
I don't follow how this would help.

https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/ca16c480a494311858176e97e785194b5ca234a3/instrumentation/elasticsearch/elasticsearch-rest-common/library/src/main/java/io/opentelemetry/instrumentation/elasticsearch/rest/internal/ElasticsearchRestInstrumenterFactory.java#L28 doesn't use `HttpCommonAttributesGetter` - can you help?

Reply:
I meant that `HttpCommonAttributesGetter` uses a single method whereas here you are using 2 methods.